package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "react-calculator",
+  "version": "1.0.0",
+  "description": "## Instructions In the initial commit to this project you have been provided with only an ESLint config for a React project. Your task is to create an interactive React Calculator page that responds to user input.",
+  "main": ".eslintrc.js",
+  "scripts": {
+    "dev": "./node_modules/.bin/webpack --progress --inline --config webpack.dev.js",
+    "lint": "./node_modules/.bin/eslint --format codeframe --ext jsx ."
```

Comment:
This command is more helpful when you actually run it

![image](https://user-images.githubusercontent.com/5775716/83074335-f6aae480-a03f-11ea-9580-cbd4596d8d37.png)

it would have found these issues for you if you had

![image](https://user-images.githubusercontent.com/5775716/83074382-0cb8a500-a040-11ea-8325-9d7b29b23138.png)